bedevere/mailer.py

```diff
@@ -0,0 +1,67 @@
+import email.message
+import email.utils
+import os
+
+from gidgethub import routing
+
+
+router = routing.Router()
+
+ALLOWED_BRANCHES = ["2.7", "3.5", "3.6", "master"]
+SENDER = os.environ.get("SENDER_EMAIL", "sender@example.com")
+RECIPIENT = os.environ.get("RECIPIENT_EMAIL", "recipient@example.com")
+
+
+def get_diff_stat(commit):
+    files = {
+        "A": commit["added"],
+        "D": commit["removed"],
+        "M": commit["modified"],
+    }
+    result = []
+    for key, file_list in files.items():
+        if file_list:
+            result.append("\n".join(f"{key} {f}" for f in file_list))
+    return "\n".join(result)
+
+
+def build_message(commit, **kwargs):
+    branch = kwargs.get("branch")
+    diff_stat = kwargs.get("diff_stat")
+    unified_diff = kwargs.get("unified_diff")
+    template = f"""\
```

Comment:
I tried that first, but for some reason, it didn't work with an f-string. Without `textwrap.dedent()`:

```
'\n    https://github.com/fayton/cpython/commit/2d420b342509e6c2b597af82ea74c4cbb13e2abd\n    commit: 2d420b342509e6c2b597af82ea74c4cbb13e2abd\n    branch: 3.5\n    author: cbiggles <berker.peksag+cbiggles@gmail.com>\n    committer: Berker Peksag <berker.peksag@gmail.com>\n    date: 2017-02-08T15:37:50+03:00\n    summary:\n\n    Update .gitignore\n(cherry picked from commit 9d9ed0e5cceef45fd63dc1f7b3fe6e695da16e83)\n\n    files:\n    M .gitignore\n\n    diff --git a/.gitignore b/.gitignore\nindex c2b4fc703f7..e0d0685fa7d 100644\n--- a/.gitignore\n+++ b/.gitignore\n@@ -93,3 +93,4 @@ htmlcov/\n Tools/msi/obj\n Tools/ssl/amd64\n Tools/ssl/win32\n+foo\n\n    '
```

With `textwrap.dedent()`:

```
'\n    https://github.com/fayton/cpython/commit/2d420b342509e6c2b597af82ea74c4cbb13e2abd\n    commit: 2d420b342509e6c2b597af82ea74c4cbb13e2abd\n    branch: 3.5\n    author: cbiggles <berker.peksag+cbiggles@gmail.com>\n    committer: Berker Peksag <berker.peksag@gmail.com>\n    date: 2017-02-08T15:37:50+03:00\n    summary:\n\n    Update .gitignore\n(cherry picked from commit 9d9ed0e5cceef45fd63dc1f7b3fe6e695da16e83)\n\n    files:\n    M .gitignore\n\n    diff --git a/.gitignore b/.gitignore\nindex c2b4fc703f7..e0d0685fa7d 100644\n--- a/.gitignore\n+++ b/.gitignore\n@@ -93,3 +93,4 @@ htmlcov/\n Tools/msi/obj\n Tools/ssl/amd64\n Tools/ssl/win32\n+foo\n\n'
```